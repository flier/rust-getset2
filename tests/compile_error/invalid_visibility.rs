use getset2::Getter;

#[derive(Getter)]
pub struct Struct {
    #[get("pub(other)")]
    field: usize,
}

fn main() {}
