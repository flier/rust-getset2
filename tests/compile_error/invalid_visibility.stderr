error: invalid #[get(..)] attribute, for the `pub` attribute, expected one of: `self`, `super`, `crate`, `in`
 --> tests/compile_error/invalid_visibility.rs:5:5
  |
5 |     #[get(pub(other))]
  |     ^
